events/eventtype.go

```diff
@@ -78,6 +79,9 @@ const (
 	EventPGOUpdateConfig    = "PGOUpdateConfig"
 	EventPGODeleteNamespace = "PGODeleteNamespace"
 	EventPGOCreateNamespace = "PGOCreateNamespace"
+
+	EventStandbyEnabled  = "StandbyEnabled"
+	EventStandbyDisabled = "StandbyDisabled"
```

Comment:
PEr earlier comments we may be able to eliminate some of these 😄

Reply:
Now that shutdowns occur asynchronously with a pgcluster update, the shutdown event remains but has been moved from the API server to the Operator.

Reply:
Enabling/disabling standby occurs asynchronously, so leaving these events.